samples/apps/copilot-chat-app/scripts/1_install-reqs.cmd

```diff
@@ -0,0 +1,5 @@
+:: Install chocolatey
+@"%SystemRoot%\System32\WindowsPowerShell\v1.0\powershell.exe" -NoProfile -InputFormat None -ExecutionPolicy Bypass -Command "[System.Net.ServicePointManager]::SecurityProtocol = 3072; iex ((New-Object System.Net.WebClient).DownloadString('https://community.chocolatey.org/install.ps1'))" && SET "PATH=%PATH%;%ALLUSERSPROFILE%\chocolatey\bin"
```

Comment:
All Windows machines have PowerShell these days - have we considered using PowerShell scripts?